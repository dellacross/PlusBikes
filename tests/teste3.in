10
100 100
67-----------------------------------------------------------------------------------------------*f*
98-----------------------------------------------------------------------------------------------igh
**-----------------------------------------------------------------------------------------------*j*
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
**-----------------------------------------------------------------------------------------------***
52-----------------------------------------------------------------------------------------------***
13----------------------------****---------------------------------------------------------------dbe
40*****************************--****************************************************************c*a
100 99 10 99 20 99 10 99 10 22
100 99 10 99 20 99 10 99 10 22
100 99 10 99 20 99 10 99 10 22
100 99 10 99 20 99 10 99 10 22
100 99 10 99 20 99 10 99 10 22
100 99 10 99 20 99 10 99 10 22
100 99 10 99 20 99 10 99 10 22
100 99 10 99 20 99 10 99 10 22
100 99 10 99 20 99 10 99 10 22
100 99 10 99 20 99 10 99 10 22