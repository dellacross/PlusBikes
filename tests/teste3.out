a 3
b 5
c 0
d 1
e 7
f 6
g 2
h 8
i 9
j 4
